src/garage/experiment/deterministic.py

```diff
@@ -4,6 +4,7 @@
 import warnings
 
 import numpy as np
+import tensorflow_probability as tfp
```

Comment:
please make sure this doesn't crash garage if TF isn't installed in the environment. we are working towards making TF optional.